backend/drive/drive.go

```diff
@@ -542,6 +543,8 @@ type Fs struct {
 	exportExtensions []string           // preferred extensions to download docs
 	importMimeTypes  []string           // MIME types to convert to docs
 	isTeamDrive      bool               // true if this is a team drive
+
+	configMapper     configmap.Mapper   // TODO: Figure a way around this
```

Comment:
Any way to avoid this and still be able to reference the mapper for creating the oAuthClient?